simulator/mips/t/unit_test.cpp

```diff
@@ -2413,3 +2409,292 @@ TEST_CASE( "MIPS32_instr: mult dump")
     instr.execute();
     CHECK( instr.string_dump() == "{0}\tmult $s1, $t1\t [ $lo = 0x0, $hi = 0x40000000 ]" );
 }
+
+TEST_CASE( "MIPS32_instr: sub")
```

Comment:
Could you please keep tests sorted in alphabet order?

Reply:
Ok